amq-streams/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Edit the CSV to disable FIPS_MODE
+  kubernetes.core.k8s:
+    state: patched
+    namespace: "{{ csv.metadata.namespace }}"
```

Comment:
`csv` fact not set in opcap-ansible.

we can use `install_namespace` here.

Reply:
Fixed.